src/plugins/code-loader.js

```diff
@@ -47,27 +43,54 @@ const plugin = (options) => {
         const startTag = customStartTag ?? "<start_here>";
         const endTag = customEndTag ?? "<end_here>";
 
-        // Split to only keep lines between "start_here" and "end_here"
-        const lines = fileContent.split(startTag).pop().split(endTag).shift();
+        let lines = fileContent.split(startTag).pop().split(endTag).shift().split('\n').slice(1,-1);
+        let finalLines = [];
+
+        if (markNumber) {
+            const markStartTag = `// <mark_${markNumber}>`;
+            const markEndTag = `// </mark_${markNumber}>`;
```

Comment:
for more clarity could this be <mark(number)> and </mark(number)>?

Reply:
Also this could perhaps accept a string too?

Reply:
It should work with string as well so maybe the underscore helps with keeping it consistent with the `// <start_here>` tags.
I want to change those start and end tags as well to be able to define them by:
`<PATH_TO_CODE>#customtag?number`
so that you dont every time need to say `#<start_tag>-<end_tag>` in those urls because that works badly in combination with other  html tags in the document.
I'll do that in a next PR.